login.go

```diff
@@ -155,6 +164,10 @@ func doTokenRequest(ctx context.Context, client *ConductoroneAPI, clientID strin
 				continue
 			}
 
+			if errResp.ErrorType == "slow_down" {
```

Comment:
hmm, we should probably do some real backoff here

Reply:
https://datatracker.ietf.org/doc/html/rfc8628#section-3.5

Updated this to reflect the RFC